src/frontend/src/server.rs

```diff
@@ -41,19 +41,26 @@ use crate::prometheus::PrometheusOptions;
 
 pub(crate) struct Services;
 
+const GRPC_SERVER: &str = "GRPC_SERVER";
+const MYSQL_SERVER: &str = "MYSQL_SERVER";
+const POSTGRES_SERVER: &str = "POSTGRES_SERVER";
+const OPENTSDB_SERVER: &str = "OPENTSDB_SERVER";
+const HTTP_SERVER: &str = "HTTP_SERVER";
+const PROM_SERVER: &str = "PROM_SERVER";
```

Comment:
Can you make these names returned by the `Server` trait? I think that makes codes more concise.